docs/user/interfaces/asyncqueryinterface.rst

```diff
@@ -35,6 +35,15 @@ The system relies on the default AWS credentials chain for making calls to the E
 *  ``applicationId``, ``executionRoleARN`` and ``region`` are required parameters.
 *  ``sparkSubmitParameter`` is an optional parameter. It can take the form ``--conf A=1 --conf B=2 ...``.
```

Comment:
https://github.com/noCharger/sql/blob/patch-1/docs/user/interfaces/asyncqueryinterface.rst

The formatting doesn't seem right. Can you fix that.

Reply:
There's a new commit https://github.com/opensearch-project/sql/commit/d0c9f366c8eda56e9fcdecb3470b1c4c4ef35056. Is this the one you are looking at?